tests/unit_tests/main_test.cpp

```diff
@@ -84,6 +84,8 @@ int main(int argc, char** argv) {
     for (auto plat : platforms) {
         if (!plat.is_host()) {
             auto plat_devs = plat.get_devices();
+            std::cout << "FOUND " << plat_devs.size() << " DEVICES "
+                      << "\n";
```

Comment:
Is it better to print it as "Available devices: devices names"? I'm not sure just number of devices is useful information without knowing what exact devices they are

Reply:
Nice catch :) I will remove it completely. Or you think we can benefit from having devices names printed?

Reply:
removed completely (otherwise it's soliciting testing log with the same message multiple times)